modules/bgg/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/bgg",
+	"url": "https://www.dndbeyond.com/sources/dnd/bgg",
```

Comment:
This URL is valid, but redirects to `https://www.dndbeyond.com/sources/dnd/bgg/baldurs-gate-gazetteer`